.github/workflows/init.yaml

```diff
@@ -43,6 +43,10 @@ on:
         description: "The name of the project to be released"
         value: ${{ jobs.init.outputs.project }}
 
+concurrency:
```

Comment:
The init yaml is integrated into control.yml therefore I don't think this is necessary here as it should be controlled by control.yml. See the on workflow_call.